src/core/LanguageMenu/LanguageMenu.md

```diff
@@ -0,0 +1,31 @@
+```js
+import {
+  LanguageMenu,
+  LanguageMenuItem
+} from 'suomifi-ui-components';
+
+<LanguageMenu
+  buttonText="In English (EN)"
+  aria-label="Change language. Current language is English"
```

Comment:
`aria-selected` or `aria-current` do not seem to work with role="menu" and "menuitem". So I thought maybe it's enough that the current language is indicated when arriving to the menu toggle button? Users with vision also see this information so screen reader users should too

Reply:
This does move the responsibility to announce current language to the library user. Maybe more realistic example would be that the  value is same that it is in buttonText, so:
ria-label="Change language. Current language In English (EN)"

Reply:
Documentation should also direct clearly to use aria-label to also announce the current selected language. 🤔